tasks/update-code/copy_zipped.yml

```diff
@@ -0,0 +1,29 @@
+---
+
+- name: ANSISTRANO | SCP_ZIP | Create release folder
+  file:
+    state: directory
+    path: "{{ ansistrano_release_path.stdout }}"
+
+- name: ANSISTRANO | SCP_ZIP | Set archived file
+  set_fact:
+    ansistrano_archived_file: "{{ ansistrano_release_path.stdout }}/data.gz"
+    work_file: "/tmp/ansistrano.gz"
+
+- name: ANSISTRANO | SCP_ZIP | Zip source file
+  command: tar czfh {{work_file}} {% for fn in ansistrano_zip_exclude %} --exclude {{fn}} {% endfor %} -C {{ ansistrano_deploy_from }} .
```

Comment:
CS in {{ sth }}

Reply:
Also, please refactor this to use archive module http://docs.ansible.com/ansible/latest/archive_module.html

Reply:
I already review using archive module. but archive module can't do this...

for example...
TARGET_FOLDER
  SUB1
     SUB1_SUB1
  SUB2
and ansistrano_deploy_from=TARGET_FOLDER
we can't exlcude SUB1_SUB1 via archive module. I viewed archive module code and understand why it's can't

I thinks I archive my purpose by using archive module. upgrade archive module first. ;-)